### DISCUSSION TITLE
bitcoin node and ord wallet on different machines?

### DISCUSSION THREAD

**Discussion by punk3700** [ordinals/ord#1584] [General]
is it possible to run bitcoin full node on a server and run my ord wallet on my local computer?

thanks!

**Comment by casey** [comment]
Yup, it is! Right now this is difficult, because the only way ord can auth against bitcoin is using the cookie file. Copy the `.cookie` file from your bitcoind data dir on another machine to your local machine, and run ord with `--cookie-file path/to/cookie/file`.
